Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -43,11 +43,16 @@ public class InteractableTests : BasePlayModeTests
         private static readonly Quaternion DefaultRotation = Quaternion.LookRotation(Vector3.up);
         private static readonly Quaternion DefaultRotationToggle = Quaternion.LookRotation(Vector3.forward);
 
+        private static InputSimulationService inputSimulationService = null;
+        private const int MOVE_HAND_NUM_STEPS = 32;
```

Comment:
>MOVE_HAND_NUM_STEPS [](start = 26, length = 19)

Nit: I think MoveHandNumSteps (all caps isn't something I've seen in our codebase)